### PR TITLE
Update requires

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: scl-rubygem-GEM.spec.template [hunk 1]
```diff
@@ -15,6 +15,11 @@ URL:     <%= spec.homepage %>
 <% end -%>
 Source0: <%= download_path %>%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+<% for d in spec.runtime_dependencies -%>
+Requires: %{?scl_prefix}rubygem(d.name)
```
Comment (ekohl) [inline scl-rubygem-GEM.spec.template]: Shouldn't this include the version requirement too?

File: packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec [hunk 2]
```diff
@@ -6,13 +6,18 @@
...
 Group:   Development/Languages
 License: MIT
 URL:     http://rubyonrails.org
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(nio4r)
+Requires: %{?scl_prefix}rubygem(websocket-driver)
+Requires: %{?scl_prefix}rubygem(actionpack) = %{version}
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec]: Is there logic to replace this exact version with the macro?

Comment (ehelms) [reply packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec]: Logic? RPM should handle it for us.

File: scl-rubygem-GEM.spec.template (hunk 1 above)
Comment (ehelms) [reply scl-rubygem-GEM.spec.template]: Take a look now at the template. I attempted my best at updating it to account for those special cases.

File: packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec (hunk 2 above)
Comment (ekohl) [reply packages/tfm-ror51-rubygem-actioncable/tfm-ror51-rubygem-actioncable.spec]: I mean that the gemspec might include version 5.1.4 but here you have the (better) macro.

File: scl-rubygem-GEM.spec.template [hunk 3]
```diff
@@ -15,6 +15,15 @@ URL:     <%= spec.homepage %>
 <% end -%>
 Source0: <%= download_path %>%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+<% for d in spec.runtime_dependencies -%>
+<% if d.include?('active') -%>
+Requires: %{?scl_prefix}rubygem(d.name) = %{spec.version}
```
Comment (ekohl) [inline scl-rubygem-GEM.spec.template]: Should this be ```%{version}```?

File: packages/tfm-ror51-rubygem-rails/tfm-ror51-rubygem-rails.spec [hunk 4]
```diff
@@ -6,13 +6,26 @@
...
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(actionmailer) = %{version}
+Requires: %{?scl_prefix}rubygem(actionpack) = %{version}
+Requires: %{?scl_prefix}rubygem(actionview) = %{version}
+Requires: %{?scl_prefix}rubygem(activejob) = %{version}
+Requires: %{?scl_prefix}rubygem(activemodel) = %{version}
+Requires: %{?scl_prefix}rubygem(activerecord) = %{version}
+Requires: %{?scl_prefix}rubygem(activesupport) = %{version}
+Requires: %{?scl_prefix_ruby}rubygem(bundler) >= 1.3.0
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-rails/tfm-ror51-rubygem-rails.spec]: I don't think we can really include this in the template but I think I can live with that.

File: scl-rubygem-GEM.spec.template (hunk 3 above)
Comment (ehelms) [reply scl-rubygem-GEM.spec.template]: Yes. good catch, updated

**Comment by ehelms** [comment]
[test copr]

File: packages/tfm-ror51-rubygem-railties/tfm-ror51-rubygem-railties.spec [hunk 5]
```diff
@@ -6,17 +6,24 @@
...
+Release: 2%{?dist}
 Summary: Tools for creating, working with, and running Rails applications
 Group:   Development/Languages
 License: MIT
 URL:     http://rubyonrails.org
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}rubygem(thor)
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-railties/tfm-ror51-rubygem-railties.spec]: Should be ```%{?scl_prefix}rubygem(thor)```

```
20:54:15 package: tfm-ror51-rubygem-railties-5.1.4-2.el7.centos.noarch from scratch
20:54:15   unresolved deps: 
20:54:15      rh-ruby24-rubygem(thor)
```

**Comment by ehelms** [comment]
[test copr]

**Pull request merged**